Tools/WebKitTestRunner/InjectedBundle/AccessibilityTextMarkerRange.cpp

```diff
@@ -70,5 +70,10 @@ JSClassRef AccessibilityTextMarkerRange::wrapperClass()
 {
     return JSAccessibilityTextMarkerRange::accessibilityTextMarkerRangeClass();
 }
-    
+
+AccessibilityTextMarkerRange* toTextMarkerRange(JSObjectRef object)
+{
+    return static_cast<AccessibilityTextMarkerRange*>(JSObjectGetPrivate(object));
```

Comment:
Perhaps there is a better way of doing this, like by directly using `toAccessibilityTextMarkerRange` from `JSAccessibilityTextMarkerRange.h` at the call site, but I don't see any of the generated JS wrapper classes being included in files other than in the implementation of the wrapped classes.